mobile/src/components/uam/groups-table/groups-table.tsx

```diff
@@ -22,8 +22,10 @@ const GroupsTable: FC = () => {
 
   const { items, total } = useAppSelector((state) => state.uam.groups);
   const { page, handlePageChange } = usePagination();
+
   const handleGroupsItemDelete = (groupId: number): void => {
     dispatch(uamActions.deleteGroup({ id: groupId }));
+    dispatch(authActions.loadCurrentUser());
```

Comment:
We should check if user was in the group and dispatch loadCurrentUser inside delete group action